Travel Scheduler/Test/Commute.m

```diff
@@ -0,0 +1,51 @@
+//
+//  Commute.m
+//  Travel Scheduler
+//
+//  Created by gilemos on 7/20/19.
+//  Copyright © 2019 aliu18. All rights reserved.
+//
+
+#import "Commute.h"
+#import "APIManager.h"
+#import "Step.h"
+
+@implementation Commute
+
+-(instancetype)initWithOrigin:(NSString *)originId toDestination:(NSString *)destinationId withDepartureTime:(int)departureTime{
+    self = [self init];
+    self.departureTimeInSecondsSince1970 = departureTime;
+    [[APIManager shared]getCommuteDetailsFromOrigin:originId toDestination:destinationId withDepartureTime:departureTime withCompletion:^(NSArray *commuteInfoArray, NSError *error) {
+        if(commuteInfoArray) {
+            [self initAllProperties];
+            [self buildCommuteWithDictionary:commuteInfoArray[0]];
+            [self makeArrayOfStepsWithRootDictionary:commuteInfoArray[0]];
+        }
+        else {
+            NSLog(@"did not work snif");
+        }
+    }];
+    return self;
+}
+
+-(void)initAllProperties {
```

Comment:
I would avoid names with `init` unless they are public initialization APIs. Name it something else like `createAllProperties` or anything else that doesn't have `init`